### DISCUSSION TITLE
notes/12.0.2/client/wn_1202_other_enhancements

### DISCUSSION THREAD

**Discussion by giscus** [HCL-TECH-SOFTWARE/HCLSoftware-Documentation-Home#94] [Q&A]
# notes/12.0.2/client/wn_1202_other_enhancements

The following are general enhancements for Notes 12.0.2.

https://help.hcl-software.com/notes/12.0.2/client/wn_1202_other_enhancements.html

**Comment by ajithtpillai** [comment]
Link  not accessible -    'Choose Microsoft Edge as Basic client internet browser option' - https://help.hcl-software.com/domino/12.0.2/admin/wn_edgeinpolicy.html

**Reply by digihernandez** [reply]
Fix. The link was updated: https://help.hcl-software.com/domino/12.0.2/admin/wn_configuration_features.html
